contracts/event/MultipleResultsEvent.sol

```diff
@@ -562,4 +519,139 @@ contract MultipleResultsEvent is NRC223Receiver, Ownable {
         uint increment = _thresholdPercentIncrease.mul(currentThreshold).div(100);
         return currentThreshold.add(increment);
     }
+
+    /// @notice Calculates the tokens returned for a non-Invalid final result.
+    /// @param better Address to calculate winnings for.
+    /// @return Amount of tokens that will be returned.
+    function calculateNormalWinnings(
+        address better)
+        private
+        view
+        returns (uint)
+    {
+        // Calculate user's winning bets
+        uint myWinningBets;
+        uint myWinningVotes;
+        for (uint8 i = 0; i <= _currentRound; i++) {
+            uint bets = _eventRounds[i].balances[_currentResultIndex].bets[better];
+            if (i == 0) {
+                myWinningBets = myWinningBets.add(bets);
+            } else {
+                myWinningVotes = myWinningVotes.add(bets);
+            }
+        }
+
+        // Calculate bet round totals
+        uint betRoundWinnersTotal =
+            _eventRounds[0].balances[_currentResultIndex].total;
+        uint betRoundLosersTotal;
+        for (uint8 i = 0; i < _numOfResults; i++) {
+            if (i != _currentResultIndex) {
+                betRoundLosersTotal = 
+                    betRoundLosersTotal.add(_eventRounds[0].balances[i].total);
+            }
+        }
+
+        // Calculate user's winning amount for bet round
+        uint maxPercent = 100;
+        uint betRoundWinningAmt;
+        if (myWinningBets > 0 && betRoundWinnersTotal > 0) {
+            uint percentage = maxPercent.sub(_arbitrationRewardPercentage);
+            betRoundWinningAmt =
+                betRoundLosersTotal
+                .mul(percentage)
+                .div(maxPercent)
+                .mul(myWinningBets)
+                .div(betRoundWinnersTotal);
+        }
+
+        // Calculate all vote rounds totals
+        uint voteRoundsWinnersTotal;
+        uint voteRoundsLosersTotal;
+        for (uint8 i = 1; i <= _currentRound; i++) {
+            for (uint8 j = 0; j < _numOfResults; j++) {
+                uint total = _eventRounds[i].balances[j].total;
+                if (j == _currentResultIndex) {
+                    voteRoundsWinnersTotal = voteRoundsWinnersTotal.add(total);
+                } else {
+                    voteRoundsLosersTotal = voteRoundsLosersTotal.add(total);
+                }
+            }
+        }
+
+        // Calculate user's winning amount for vote rounds
+        uint voteRoundsWinningAmt;
+        if (myWinningVotes > 0 && voteRoundsWinnersTotal > 0) {
+            voteRoundsWinningAmt =
+                betRoundLosersTotal
+                .mul(_arbitrationRewardPercentage)
+                .div(maxPercent)
+                .add(voteRoundsLosersTotal)
+                .mul(myWinningVotes)
```

Comment:
Just to clarify, here we we will call `add` first, right?
Can we write in 
```
voteRoundsWinningAmt =
                (betRoundLosersTotal
                .mul(_arbitrationRewardPercentage)
                .div(maxPercent)
                .add(voteRoundsLosersTotal))
                .mul(myWinningVotes)
                .div(voteRoundsWinnersTotal);
``` to clarify it

Reply:
its not necessary since doing SafeMath operations are sequential. each call will return the updated amount. can check SafeMath contract.